gssapi/tests/test_raw.py

```diff
@@ -424,16 +434,21 @@ def test_store_cred_acquire_cred(self):
         self.assertIsNotNone(deleg_creds)
 
         store_res = gb.store_cred(deleg_creds, usage='initiate',
+                                  mech=gb.MechType.kerberos,
                                   set_default=True, overwrite=True)
         self.assertIsNotNone(store_res)
-        self.assertEqual(store_res.usage, "initiate")
-        self.assertIn(gb.MechType.kerberos, store_res.mechs)
+
+        if self.realm.provider.lower() != 'heimdal':
+            # Heimdal return this info properly
+            self.assertEqual(store_res.usage, "initiate")
+            self.assertIn(gb.MechType.kerberos, store_res.mechs)
 
         deleg_name = gb.inquire_cred(deleg_creds).name
         acq_resp = gb.acquire_cred(deleg_name, usage='initiate')
         self.assertIsNotNone(acq_resp)
 
     @ktu.gssapi_extension_test('cred_store', 'credentials store')
+    @ktu.krb_provider_test(['mit'], 'FIXME')
```

Comment:
I'd prefer not to commit "FIXME"s into the project :)

Can you adjust this to mention something about Heimdal not implementing the cred store extensions instead?

Reply:
Ah this was a mistake, I was testing out an issue on macOS and forgot to remove this once I figured it out. Thanks for the pickup.